application/app/src/main/resources/application.yaml

```diff
@@ -25,8 +25,8 @@ api:
     service_search: ${API_SERVICE_SEARCH_ENDPOINT}
   cors:
     allowedorigins: ${ALLOWED_ORIGINS}
-  version: 1.0
-  title: NHS Fuzzy Search Application Interface
+  version: 1.0.0
+  title: NHS Fuzzy Search API
   description: NHS Fuzzy search APIs provide NHS Services information to the users
```

Comment:
Description: Provides service information given a set of fuzzy search criteria.

Reply:
Thanks Jon, let's discuss more details on the criteria in swagger description.